addons/medical/functions/fnc_addDamageToUnit.sqf

```diff
@@ -40,14 +40,14 @@ if (!isNull _instigator) then {
 };
 
 #ifdef DEBUG_TESTRESULTS
-private _startDmg = +(_unit getVariable [QGVAR(bodyPartDamage), [-1]]);
+private _startDmg = +(_unit getVariable [QGVAR(medical,bodyPartDamage), [-1]]);
```

Comment:
Why is QGVAR(medical,bodyPartDamage) used in here? should not it be QEGVAR(medical,bodyPartDamage) or since we are in medical module leave it as QGVAR(bodyPartDamage)?

Reply:
Good catch! For the time being EGVAR is fine as things may be moved around a bit more and it saves them breaking